tests/baselines/reference/optionalMethods.js

```diff
@@ -128,11 +128,11 @@ interface Foo {
 declare function test1(x: Foo): void;
 declare class Bar {
     d: number;
-    e: number;
+    e: number | undefined;

```

Comment:
This is incorrect -- because there was a non-`undefined` initializer, the property should not include `undefined`